ArchUnitNET/Fluent/Syntax/Elements/GivenObjectsConjunctionWithoutBecause.cs

```diff
@@ -0,0 +1,31 @@
+﻿using ArchUnitNET.Domain;
+using static ArchUnitNET.Fluent.Syntax.ActivatorHandler;
+
+namespace ArchUnitNET.Fluent.Syntax.Elements
+{
+    public abstract class
+        GivenObjectsConjunctionWithoutBecause<TGivenRuleTypeThat, TRuleTypeShould, TRuleType> : SyntaxElement<TRuleType>
```

Comment:
WithoutBecause is not a good name. How about "WithoutReason"?

Reply:
Renamed it to GivenObjectsConjunctionWithReason, as it is instantiated if `Because()` was already used in the rule.